runtime/lua/man.lua

```diff
@@ -1,14 +1,14 @@
-require('vim.compat')
+require "vim.compat"
```

Comment:
if we are going to enforce one style of single line strings, why not ' instead of " ? I find the former easier on the eye.

Reply:
I don't have a strong preference, @tjdevries you?

Reply:
FWIW, I prefer single quotes as well (don't know why; probably just Stockholm Syndrome by now...)

Reply:
I don't have a strong opinion here . But I'm also inclined towards single quotes.